src/main/features/dashboard/views/macro/caseStatus.njk

```diff
@@ -271,7 +275,11 @@
   {% elif status === ClaimStatus.SETTLEMENT_AGREEMENT_REJECTED %}
     {{ settlementAgreementRejectedForDefendantDashboard() }}
   {% elif status === ClaimStatus.CLAIMANT_REJECTED_DEFENDANT_DEFENCE_NO_DQ %}
```

Comment:
in claimStatus I cas see 'CLAIMANT_REJECTED_DEFENDANT_DEFENCE_NO_DQ' is been removed, do we still need this check?